Small update to release note 1.6 (#7373)
istio/istio.io: content/en/news/releases/1.6.x/announcing-1.6/change-notes/index.md

- ***Removed*** built in Istio configurations from the installation, including the Gateway, `VirtualServices`, and mTLS settings.
- ***Added*** a new profile, called `preview`, allowing users to try out new experimental features that include WASM enabled telemetry v2.
- ***Added*** `istioctl install` command as a replacement for `istioctl manifest apply`.

> - ***Added*** istiod-remote chart to allow users to [experiement istiod manages remote data plane] (https://github.com/istio/istio/wiki/Central-Istiod-manages-remote-data-plane).

Reviewer: Maybe something like:
```suggestion
- ***Added*** istiod-remote wiki describing how to [experiment with a central Istiod managing a remote data plane] (https://github.com/istio/istio/wiki/Central-Istiod-manages-remote-data-plane).
```

> - ***Added*** istiod-remote chart to allow users to [experiment with a central Istiod managing a remote data plane] (https://github.com/istio/istio/wiki/Central-Istiod-manages-remote-data-plane).

Reviewer 2: ```suggestion
- ***Added*** istiod-remote chart to allow users to [experiment with a central Istiod managing a remote data plane](https://github.com/istio/istio/wiki/Central-Istiod-manages-remote-data-plane).
```

Reviewer 2: You need to remove the space to fix the lint issue.
